CMakeLists.txt

```diff
@@ -3,4 +3,7 @@ project(MyApplication)
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/modules/" ${CMAKE_MODULE_PATH})
 
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(GTKMM gtkmm-3.0)
```

Comment:
This could be inside `src/CMakeLists.txt` to keep it localized.

Reply:
I thought it was odd that the official guide recommends putting it in the top level cmake file, too. I'm pretty sure it works either way, so I'll move it.